fe/src/test/java/org/apache/doris/load/routineload/RoutineLoadManagerTest.java

```diff
@@ -56,8 +57,6 @@
 import mockit.MockUp;
 import mockit.Mocked;
 
-import static mockit.Deencapsulation.invoke;
```

Comment:
Static import must at the top of imports？

Reply:
Fixed